source/contracts/reporting/Market.sol

```diff
@@ -233,24 +225,23 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
             return false;
         }
 
-        if (getUniverse().getForkingMarket() == this) {
+        if (getForkingMarket() == this) {
             tentativeWinningPayoutDistributionHash = getWinningPayoutDistributionHashFromFork();
         }
 
-        require(tentativeWinningPayoutDistributionHash != bytes32(0));
-
         finalPayoutDistributionHash = tentativeWinningPayoutDistributionHash;
         finalizationTime = block.timestamp;
         transferIncorrectDisputeBondsToWinningReportingToken();
+        // The validity bond is paid to the owner in any indeterminate outcome and the reporting window otherwise
+        doFeePayout(!isIndeterminate(), validityBondAttoeth);
```

Comment:
Wording: I believe we have officially decided to call it `valid` and `invalid` and strike `indeterminate` from the record.  Feel free to address naming in a separate PR, but we should try to get some consistency on that in the long run.

Reply:
Also, I would prefer a function named `isValid` rather than `isInvalid` so we can avoid double negatives like this.

Reply:
I'll add it here. I was fighting calling it invalid the whole time anyway so that'll scratch an itch.